locale/hu.json

```diff
@@ -107,8 +107,8 @@
                 },
                 "superjump": {
                     "title": "Super Jump",
-                    "label": "Toggle super jump mode, the player will also run faster",
-                    "success": "Super Jump enabled"
+                    "label": "Magas ugrás és gyors futás",
+                    "success": "Super Jump engedélyezve"
```

Comment:
`bekapcsolva` maybe?

Reply:
azért írtam engedélyezve-re, mert ha megnézed a fölötte lévőket azok is úgy vannak, mondom akkor legyen rendszer nézzen ki ugyan úgy.

Reply:
jogos